<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StokBilgileriFormu</class>
 <widget class="QDialog" name="StokBilgileriFormu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ürün Bilgileri</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../materials.qrc">
    <normaloff>:/icons/Icons/cart_add.png</normaloff>:/icons/Icons/cart_add.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
	
	font: 10pt &quot;Calibri&quot;;
}
QDateEdit {
qproperty-displayFormat: &quot;dd.MM.yyyy&quot;;
qproperty-calendarPopup: true;
}
QPushButton#pb_Kaydet{
qproperty-text: &quot;Kaydet&quot;;
qproperty-default: true;
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>350</y>
     <width>278</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pb_TTemizle">
      <property name="text">
       <string>Tümünü Temizle</string>
      </property>
      <property name="icon">
       <iconset resource="../materials.qrc">
        <normaloff>:/icons/Icons/trash-empty.png</normaloff>:/icons/Icons/trash-empty.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_Kaydet">
      <property name="text">
       <string>Kaydet</string>
      </property>
      <property name="icon">
       <iconset resource="../materials.qrc">
        <normaloff>:/icons/Icons/Ok.png</normaloff>:/icons/Icons/Ok.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_Cikis">
      <property name="text">
       <string>Çıkış</string>
      </property>
      <property name="icon">
       <iconset resource="../materials.qrc">
        <normaloff>:/icons/Icons/No.png</normaloff>:/icons/Icons/No.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>99</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../materials.qrc">:/icons/Icons/cart_add.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Ürün Ekle</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>321</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_Adi">
      <property name="text">
       <string>Adı:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="le_Adi"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lbl_Kodu">
      <property name="text">
       <string>Kodu:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="le_Kodu"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Adeti:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Fiyatı:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Tarihi:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDateEdit" name="dateEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../materials.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pb_Cikis</sender>
   <signal>clicked()</signal>
   <receiver>StokBilgileriFormu</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>358</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_Kaydet</sender>
   <signal>clicked()</signal>
   <receiver>StokBilgileriFormu</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>358</y>
    </hint>
    <hint type="destinationlabel">
     <x>29</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>le_Adi</sender>
   <signal>textChanged(QString)</signal>
   <receiver>StokBilgileriFormu</receiver>
   <slot>degisiklikvar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>le_Kodu</sender>
   <signal>textChanged(QString)</signal>
   <receiver>StokBilgileriFormu</receiver>
   <slot>degisiklikvar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>degisiklikvar()</slot>
 </slots>
</ui>
